packages/medusa/src/api/routes/admin/orders/__tests__/get-order.js

```diff
@@ -17,13 +17,20 @@ const defaultRelations = [
   "returns.items.reason",
   "gift_cards",
   "gift_card_transactions",
+  "items",
```

Comment:
We might need to review whether adding these relations only were necessary locally. I believe @olivermrbl talked about incl. relations which are also lazy loaded causes errors.